cmd/client/client.go

```diff
@@ -13,137 +13,128 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-
-// Package client creates a tunnel client to proxy incoming connections
+//
+// This binary creates a tunnel client to proxy incoming connections
 // over a grpc transport.
-package client
+// Exmaples to use this binary with ssh's ProxyCommand option:
+// TLS:
+//		ssh -o ProxyCommand="client
+//		--tunnel_server_address=localhost:$PORT \
+//		--cert_file=$CERT_FILE \
+//		--dial_target=target1 \
+//		--dial_target_type=SSH" $USER@localhost
+// mTLS:
+// ssh -o ProxyCommand="client
```

Comment:
Whitespace to be consistent with above TLS example.

Reply:
Done.